youki_integration_test/Cargo.toml

```diff
@@ -12,12 +12,17 @@ features = ["std", "suggestions", "derive"]
 version = "=3.0.0-beta.2"
 default-features = true
 
+[workspace]
+members = [
+    "test_framework",
+]
+
 [dependencies]
 uuid = "0.8"
 rand = "0.8.0"
 tar = "0.4"
 flate2 = "1.0"
-test_framework = { version = "0.1.0", path = "../test_framework"}
+test_framework = { version = "0.1.0", path = "./test_framework"}
```

Comment:
Do we need to specify the version of test_framework?

Reply:
No, I will remove it in the update